front/tests/017-stdcm-missing-fields.spec.ts

```diff
@@ -0,0 +1,96 @@
+import type { Infra } from 'common/api/osrdEditoastApi';
+
+import {
+  allMissingFieldsKeys,
+  getFieldsLabel,
+  partialMissingFieldsKeys,
+} from './assets/constants/missing-fields';
+import { electricRollingStockName } from './assets/constants/project-const';
+import test from './logging-fixture';
+import ConsistSection from './pages/stdcm/consist-section';
+import DestinationSection from './pages/stdcm/destination-section';
+import OriginSection from './pages/stdcm/origin-section';
+import SimulationResultPage from './pages/stdcm/simulation-results-page';
+import STDCMPage from './pages/stdcm/stdcm-page';
+import { getTranslations, waitForInfraStateToBeCached } from './utils';
+import { getInfra } from './utils/api-utils';
+import readJsonFile from './utils/file-utils';
+import type { StdcmTranslations } from './utils/types';
+
+const enTranslations: StdcmTranslations = readJsonFile('public/locales/en/stdcm.json');
+const frTranslations: StdcmTranslations = readJsonFile('public/locales/fr/stdcm.json');
+
+test.describe('Verify stdcm missing fields', () => {
+  test.slow(); // Mark test as slow due to multiple steps
+  test.use({ viewport: { width: 1920, height: 1080 } });
+
+  let stdcmPage: STDCMPage;
+  let consistSection: ConsistSection;
+  let originSection: OriginSection;
+  let destinationSection: DestinationSection;
+  let simulationResultPage: SimulationResultPage;
+  let translations: StdcmTranslations;
+  let infra: Infra;
+
+  test.beforeAll('Fetch infrastructure', async () => {
+    infra = await getInfra();
+  });
+
+  test.beforeEach('Navigate to the STDCM page', async ({ page }) => {
+    [stdcmPage, consistSection, originSection, destinationSection, simulationResultPage] = [
+      new STDCMPage(page),
+      new ConsistSection(page),
+      new OriginSection(page),
+      new DestinationSection(page),
+      new SimulationResultPage(page),
+    ];
+    translations = getTranslations({ en: enTranslations, fr: frTranslations });
+
+    await page.goto('/stdcm');
+    await page.waitForLoadState('networkidle');
+    await stdcmPage.removeViteOverlay();
+
+    // Wait for infra to be in 'CACHED' state before proceeding
+    await waitForInfraStateToBeCached(infra.id);
+  });
+
+  /** *************** Test 1 **************** */
+  test('Verify missing fields warnings when launching simulation', async () => {
+    // Step 1 — verify all missing fields
+    const allMissingLabels = getFieldsLabel(allMissingFieldsKeys, translations);
```

Comment:
I believe for const we use capital cases in snake case (same for `partialMissingLabels`)
```suggestion
    const allMissingLabels = getFieldsLabel(ALL_MISSING_FIELDS_KEY, translations);
```